tests/MongoDB.Driver.Tests/Specifications/client-side-encryption/ClientSideEncryptionTestRunner.cs

```diff
@@ -226,6 +227,17 @@ private AutoEncryptionOptions ConfigureAutoEncryptionOptions(BsonDocument autoEn
         {
             var extraOptions = new Dictionary<string, object>();
 
+            if (!CoreTestConfiguration.GetTestMongdocryptd())
```

Comment:
I think this should be done inside `EncryptionTestHelper.ConfigureDefaultExtraOptions`